main.go

```diff
@@ -24,7 +24,7 @@ func main() {
 	opts := &plugin.ServeOpts{
 		Debug:        debugMode,
 		ProviderAddr: "registry.terraform.io/thousandeyes/thousandeyes",
-		ProviderFunc: thousandeyes.New(version),
+		ProviderFunc: thousandeyes.New(),
```

Comment:
is the global var `version` still needed?

Reply:
Don't think so. Looking at the comment, it seems that goreleaser did something with the version... not sure how though, since the Provider didn't use the variable 🤔